src/store/modules/aeternity.js

```diff
@@ -0,0 +1,205 @@
+/* globals Cypress */
+import {
+  MemoryAccount, Node, Universal, RpcAepp,
+} from '@aeternity/aepp-sdk/es';
+import BrowserWindowMessageConnection from '@aeternity/aepp-sdk/es/utils/aepp-wallet-communication/connection/browser-window-message';
+import Detector from '@aeternity/aepp-sdk/es/utils/aepp-wallet-communication/wallet-detector';
+import TIPPING_INTERFACE_V1 from 'tipping-contract/Tipping_v1_Interface.aes';
+import TIPPING_INTERFACE_V2 from 'tipping-contract/Tipping_v2_Interface.aes';
+import TIPPING_INTERFACE_V3 from 'tipping-contract/Tipping_v3_Interface.aes';
+import tippingContractUtil from 'tipping-contract/util/tippingContractUtil';
+import FUNGIBLE_TOKEN_CONTRACT from 'aeternity-fungible-token/FungibleTokenFullInterface.aes';
+import { BigNumber } from 'bignumber.js';
+import { IS_MOBILE_DEVICE } from '../../utils';
+
+export default {
+  state: {
+    sdk: null,
+    useSdkWallet: false,
+    useIframeWallet: false,
+    contractV1: null,
+    contractV2: null,
+    contractV3: null,
+    isTippingContractsInitialised: false,
+  },
+  mutations: {
+    setAllContracts(state, contracts) {
+      const [contractV1, contractV2, contractV3] = contracts;
+      state.contractV1 = contractV1;
+      state.contractV2 = contractV2;
+      state.contractV3 = contractV3;
```

Comment:
```suggestion
      state.contractV3 = contractV3;
      state.isTippingContractsInitialised = true;
```
`isTippingContractsInitialised` is not changed separately, so I think we can do it at the same time.

Reply:
OK, so I don't think we should keep the setTippingContractsInitialised mutation too.